lib/driverProviders/sauce.js

```diff
@@ -5,7 +5,7 @@
  */
 
 var util = require('util'),
-    log = require('../logger.js'),
+    log = require('../logger.js').Logger,
```

Comment:
If you like, you can avoid this change by continuing to use default export for the Logger symbol.
https://github.com/Microsoft/TypeScript/blob/master/doc/spec.md#11.3.4.2